src/Controller/UI/DatasetSubmissionController.php

```diff
@@ -598,7 +598,7 @@ private function defaultDistributionPoint(DatasetSubmission $datasetSubmission,
             if (null === $distributionPoint->getDataCenter()) {
                 $distributionPoint->setRoleCode(self::DEFAULT_DISTRIBUTION_POINT_ROLECODE);
                 $distributionPoint->setDataCenter($defaultDistributionContacts[0]);
-                $distributionPoint->setDistributionUrl($this->generateUrl('pelagos_homepage') . '/data/' . $udi);
+                $distributionPoint->setDistributionUrl($this->generateUrl('pelagos_homepage', [], 0) . 'data/' . $udi);
```

Comment:
For this parameter use: `UrlGeneratorInterface::ABSOLUTE_URL`
Also add a use for this.
`use Symfony\Component\Routing\Generator\UrlGeneratorInterface;`

Reply:
Also, why not generate the data landing URL?

Reply:
Made the changes